distbench_utils.cc

```diff
@@ -746,7 +746,12 @@ GetCanonicalDistributedSystemDescription(
 absl::StatusOr<TestSequence> GetCanonicalTestSequence(
     const TestSequence& sequence) {
   TestSequence ret;
-  *ret.mutable_tests_setting() = sequence.tests_setting();
+  if (sequence.tests_setting().IsInitialized()) {
```

Comment:
use has_test_setting()

Pass the test setting to a validator function, and propogate any error that it returns.